src/app/pages/send/send-crypto-asset-form/family/bitcoin/hooks/use-generate-bitcoin-tx.ts

```diff
@@ -22,7 +22,11 @@ export function useGenerateSignedBitcoinTx() {
   const networkMode = useBitcoinLibNetworkConfig();
 
   return useCallback(
-    (values: BitcoinSendFormValues, feeRate: number) => {
+    (
+      // Pick being used here to share with RPC send transfer
+      values: BitcoinSendFormValues | Pick<BitcoinSendFormValues, 'amount' | 'recipient'>,
```

Comment:
How can we change this to be something less feature specific and more generic? 

Problem here, and with initial implementation, is that we coupled generating a bitcoin tx to a specific instance of us sending a tx (via the form). This coupling grows, and here we add to the list. 

Perhaps we need to generate a tx again in another case, and we expand this list of possible allowed values. 

Better, might be to create a new interface of all possibly allowed options, removing the tie to the send form.

Reply:
Yep, I agree here. I've also mentioned to pull a lot of components out of the send form specifically to a higher level so they can be reused. Not sure when the best time is to take that on?